### DISCUSSION TITLE
Github Pack - High School Problem

### DISCUSSION THREAD

**Discussion by leopg2** [community/community#57448] [GitHub Education]
### Hi everyone,

Hello my name is Padurean Gabriel Leonard, student at Nicolae Iorga Technological High School from Negresti town in Vaslui county - Romania. I want to apply for the Github Student Developer Pack but when I fill out that form, my high school does not appear in the list and the email (leonard.padurean@lni.ro) appears as belonging to ltnibr.ro which is not my high school website/domain. Can you help me with this?
![image](https://github.com/community/community/assets/134538774/5848741d-f613-4b39-b27f-c07e0daa3e5c)

**Comment by leopg2** [comment]
up. help me please

**Comment by leopg1** [comment]
up

**Comment by leopg1** [comment]
up

**Comment by samus-aran** [comment]
Hi @leopg2 👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
